turbinia/config/recipes/triage-linux.yaml

```diff
@@ -0,0 +1,23 @@
+# Triage recipe for quick initial response (specific to Linux).
+
+globals:
+  jobs_allowlist:
+    - CronExtractionJob
+    - CronAnalysisJob
+    - PlasoJob
+    - PsortJob
+    - FileSystemTimelineJob
+
+plaso_base:
+  task: 'PlasoTask'
+
+  file_filter: [
+      '/var/log/\w+[.]log([.]\d.gz|[.]\d|$)',
+      '/var/log/audit/audit[.]log([.]\d[.]gz|[.]\d|$)',
+      '/home/.+/[.]bash_history',
```

Comment:
I'm not sure how Plaso handles specifying file filters and artifact filters at the same time, but I wonder if we should use the corresponding ForensicsArtifacts here?  I think it has more comprehensive shell history definitions at least, though, I could be wrong.

Reply:
My understanding is Plaso requires a path or regular expression, hence why I used regex here. Some of it I grabbed from ForensicArtifacts definitions, some I added myself. I will take another look and see if there are more artifacts we can add to the filter lists.

Reply:
Let me run some tests using artifact_filters and see what the outcome is.

Reply:
As discussed, using artifact_filters is the better choice but it is mutually exclusive with file_filter. Made changes and added a note in the PlasoTask about this.